README.rst

```diff
@@ -22,7 +22,8 @@ An experimental Windows binary (portable) can be downloaded from the `devbuild
 Github Action
 <https://github.com/solvcon/modmesh/actions/workflows/devbuild.yml?query=event%3Aschedule+is%3Asuccess+branch%3Amaster>`__.
 Click the Windows release run and scroll down to the "artifacts" section to
-download the ZIP file (Login is required).
+download the zip file (login to `GitHub <https://github.com/>`__ is required).
+A direct download link can be found in https://doc.solvcon.net/.
```

Comment:
Why don't we have  a direct link here. It's not user-friendly to go to the website just for finding the URL.

Reply:
@terrychan999 has tried to include a link in MD or RST, but github does not provide something allowing us to do it.  This is the "link" @terrychan999 managed to make:

```html
```

Reply:
@terrychan999 has tried to include a link in MD or RST, but github does not provide something allowing us to do it.  This is the "link" @terrychan999 managed to make:

```html

   <script async>
     async function fetchWorkflowData() {
       const baseUrl = 'https://api.github.com/repos/solvcon/modmesh/actions/runs';
       const queryParams = '?event=schedule&status=success';
       const response = await fetch(`${baseUrl}${queryParams}`);
       if (!response.ok) {
         console.error(`Failed to fetch workflow runs: ${response.statusText}`);
         return; // Exit if the initial fetch fails
       }
       const data = await response.json();
       const devbuildRuns = data.workflow_runs.filter(run => run.name === "devbuild");
       if (devbuildRuns.length === 0) {
         console.error('No successful devbuild runs found.');
         return; // Exit if no devbuild runs are found
       }
       let foundArtifact = false; // Flag to track if any valid artifacts are found
       for (const run of devbuildRuns) {
         await new Promise(resolve => setTimeout(resolve, 200)); // delay 200ms between tries to avoid rate limiting
         const artifactsResponse = await fetch(run.artifacts_url);
         if (!artifactsResponse.ok) {
           console.error(`Failed to fetch artifacts for run ID: ${run.id}`);
           continue; // Skip to the next run if the fetch fails
         }
         const artifactsData = await artifactsResponse.json();
         if (artifactsData.artifacts && artifactsData.artifacts.length > 0) {
           const artifactId = artifactsData.artifacts[0].id;
           console.log('Artifact ID:', artifactId);
           const downloadUrl = `https://github.com/solvcon/modmesh/actions/runs/${run.id}/artifacts/${artifactId}`
           console.log('Download URL:', downloadUrl);
           window.location.href = downloadUrl;
           foundArtifact = true;
           break; // Exit the loop after finding the first valid artifact
         } else {
           console.error(`No artifacts found for run ID: ${run.id}`);
         }
       }
       if (!foundArtifact) {
         console.error('No downloadable artifacts were found after checking all eligible devbuild runs.');
       }
     }
   </script>

   <p>
   An experimental pre-built Windows binary for <a
   href="https://github.com/solvcon/modmesh/">modmesh</a> is made in a zip file
   with <a
   href="https://github.com/solvcon/modmesh/actions/workflows/devbuild.yml?query=event%3Aschedule+is%3Asuccess+branch%3Amaster">
   the "devbuild" nightly runs in GitHub Actions</a>. You can scroll down to the
   "artifacts" section in the Windows release run to download it (login to <a
   href="https://github.com/">GitHub</a> is required). Or you can directly click
   <a href="javascript:void(0);" onclick="fetchWorkflowData()">the link that
   points to the nightly build</a> to download.
   </p>
```

Specifically, this part:

```html
<a
href="https://github.com/">GitHub</a> is required). Or you can directly click
<a href="javascript:void(0);" onclick="fetchWorkflowData()">the link that
points to the nightly build</a>
```

It's technically adorable.

Reply:
I made minor adjustments to Terry's original implementation by changing from `<script defer>` to `<script async>`.

Reply:
Once reaching https://github.com/solvcon/modmesh/milestone/2 we will be able to plan for a release to provide file downloading.

Reply:
I also wonder if it is possible to embed JS in a markdown page, but unfortunetly, [no](https://stackoverflow.com/questions/21340803/embed-javascript-in-github-readme-md#:~:text=As%20such%20it's%20natural%20that,domain%20is%20not%20the%20same.).

Reply:
For security it makes sense to forbid JavaScript from GitHub website.  Anyway, it's best to use officially supported feature.  I am merging this and move on.